### PR TITLE
Unindex layers with issues-Issue 307

### PR DESCRIPTION
**Pull request by zhangmingda**
Remove the index for layers in search backend, which are linked to an issue.

### CODE REVIEW

File: hypermap/aggregator/tasks.py [hunk 1]
```diff
@@ -288,6 +288,19 @@ def index_layer(self, layer_id, use_cache=False):
...
 
 @shared_task(bind=True)
+def unindex_layers_with_issues(self, use_cache=False):
+    """
+    Remove the index for layers in search backend, which are linked to an issue.
+    """
+    from hypermap.aggregator.models import Issue, Layer
+    from django.contrib.contenttypes.models import ContentType
+
+    layer_type = ContentType.objects.get_for_model(Layer)
```
Comment (capooti) [inline hypermap/aggregator/tasks.py]: I think we need also to run the code for all of the layers which are part of a service which an associated issue

**Comment by capooti** [comment]
Thanks @zhangmingda 
review my comment, push the change and I will merge it.
If you have time it would be great to have a test

File: hypermap/aggregator/tasks.py (hunk 1 above)
Comment (zhangmingda) [inline hypermap/aggregator/tasks.py]: Thanks for the comment. I will add some related codes.

**Comment by zhangmingda** [comment]
Sure. I tested the codes to unindex layers with issues manually. I will test it again after I add some new codes.

**Comment by zhangmingda** [comment]
@capooti, I tested the codes manually using solr backend. I will add some test codes a little later.

**Comment by capooti** [comment]
great!
It looks like there are unrelated commits?
.travis is already updated now, you don't need to commit it: https://github.com/cga-harvard/HHypermap/commit/9044f8a637187b98e33d36ff61744fc4236f1af5

also squash your commits to just one commit. Thanks a lot!

**Comment by zhangmingda** [comment]
I will submit a new RP later.

**Pull request closed without merging**